### PR TITLE
Clarify innerText behavior.

### LINKED ISSUE #54: Spec innerText interactions with s-v
I think we should not yield any innerText results when querying a locked subtree (similar to visibility: hidden).

### PR DESCRIPTION
**Pull request by vmpstr**
Closes #54

### CODE REVIEW

**Comment by vmpstr** [comment]
PTAL

File: index.bs [hunk 1]
```diff
@@ -301,6 +301,10 @@ Restrictions and Clarifications {#restrictions}
       step has to run at least once.
     </div>
 
+9. Neither the [=skipped=] element nor any of its subtree elements contribute
```
Comment (chrishtr) [inline index.bs:304]:
```suggestion
9. The subtree of a [=skipped=] element does not contribute
```

**Comment by vmpstr** [comment]
Yeah that absolutely makes sense.

**Comment by vmpstr** [comment]
PTAL

**Pull request merged**